### PR TITLE
Resubmit of: Add tutorial for modifying kernel boot parameters #628

### PR DESCRIPTION
**Pull request by intelkevinputnam**
Due to massive structural changes of rtd-theme merge, it makes more sense to resubmit.

Signed-off-by: Kevin Putnam <kevin.putnam@intel.com>

### CODE REVIEW

**Comment by intelkevinputnam** [comment]
Apologize for the second request. I needed to submit a new PR to deal with massive organizational changes to the documentation.

File: source/tutorials/kernel-boot-params.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+   man clr-boot-manager
+
+.. contents:: 
+    :local:
+    :depth: 1
+
+Updating the boot manager
+*************************
+
+The system must be updated and rebooted each time a changes is made
```
Comment (bwarden) [inline source/tutorials/kernel-boot-params.rst]:
```suggestion
The system must be updated and rebooted each time a change is made
```

Comment (intelkevinputnam) [reply source/tutorials/kernel-boot-params.rst]: Thanks!

File: source/tutorials/kernel-boot-params.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
+.. _kernel_boot_params:
+
+Kernel boot parameters
+######################
+
+Changing kernel boot paramters can drastically change the way the OS behaves,
```
Comment (mvincerx) [inline source/tutorials/kernel-boot-params.rst]: "parameter"

I like this intro.

File: source/tutorials/kernel-boot-params.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+.. code-block:: bash
+
+   man clr-boot-manager
+
+.. contents:: 
+    :local:
+    :depth: 1
+
+Updating the boot manager
```
Comment (mvincerx) [inline source/tutorials/kernel-boot-params.rst]: Updating -> Update

File: source/tutorials/kernel-boot-params.rst [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+Remove the "quiet" kernel boot option:
+
+.. code-block:: bash
+
+   sudo mkdir -p /etc/kernel/cmdline-removal.d
+   echo "quiet" | sudo tee /etc/kernel/cmdline-removal.d/quiet.conf
+
+Update the boot manager and reboot.
+
+Restoring default boot parameters
```
Comment (mvincerx) [inline source/tutorials/kernel-boot-params.rst]: Restoring -> Restore

**Comment by intelkevinputnam** [comment]
@mvincerx Any thoughts on what should be in a Related topics section?

**Comment by mvincerx** [comment]
I like the idea of adding  `:ref:`kernel-modules.html`` in Related topics. While the purposes are different, there's a little overlap of knowledge in working with the `clr-boot-manager`.  A user might like the convenience of that link--especially if it were a kernel developer who really wants to customize.

**Comment by intelkevinputnam** [comment]
@mvincerx Thanks for the suggestion. Added.

File: source/guides/kernel/kernel-boot-params.rst [hunk 5]
```diff
@@ -0,0 +1,81 @@
+.. _kernel_boot_params:
+
+Kernel boot parameters
+######################
+
```
Comment (puneetse) [inline source/guides/kernel/kernel-boot-params.rst]: Suggest adding an opening sentence about what a kernel boot parameter is. Right now is immediately opens with a warning without context.

Also suggest linking to https://www.kernel.org/doc/Documentation/admin-guide/kernel-parameters.txt at some point

Comment (mvincerx) [reply source/guides/kernel/kernel-boot-params.rst]: Hold until after 09-06.

**Pull request closed without merging**